File: bench/cases/csv-trim-100k/main.ts
function rowLen(line: string): number {
  const parts = line.split(',')
  let total = 0
  for (let i = 0; i < parts.length; i = i + 1) {
    const t = parts[i].trim()
    total = total + t.length
  }
  return total
}

let total = 0
const n = 100000
for (let i = 0; i < n; i = i + 1) {
  total = total + rowLen('  alpha , beta , gamma , delta , epsilon , zeta  ')
}
console.log(total)

File: bench/cases/csv-trim-100k/main.rs
fn row_len(line: &str) -> i64 {
    let mut total: i64 = 0;
    for part in line.split(',') {
        total += part.trim().len() as i64;
    }
    total
}

fn main() {
    let mut total: i64 = 0;
    let n: i64 = 100_000;
    for _ in 0..n {
        total += row_len("  alpha , beta , gamma , delta , epsilon , zeta  ");
    }
    println!("{}", total);
}
